.gitlab-ci.yml

```diff
@@ -360,11 +350,36 @@ Integration:
           - aws/rhel-9.0-nightly-x86_64
           - aws/centos-stream-9-x86_64
         INTERNAL_NETWORK: ["true"]
-      - SCRIPT:
-          - azure_hyperv_gen2.sh
```

Comment:
This test is annoying, we should restructure it. Will look into this later.

Reply:
https://github.com/osbuild/osbuild-composer/issues/2491